src/main/java/clzzz/helper/commons/util/StringUtil.java

```diff
@@ -35,7 +35,14 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
         String[] wordsInPreppedSentence = preppedSentence.split("\\s+");
 
         return Arrays.stream(wordsInPreppedSentence)
-                .anyMatch(preppedWord::equalsIgnoreCase);
+                .anyMatch(eachWord -> containsAsSubstringIgnoreCase(eachWord, preppedWord));
```

Comment:
Wouldn't containsIgnoreCase() be enough?

Reply:
🤔

Reply:
ok haha I didn't know there is such function